src/App.js

```diff
@@ -5,17 +5,25 @@ import Footer from './Footer';
 import Header from './Header';
 import beastData from './data.json';
 import SelectedBeast from './SelectedBeast';
+import HornForm from './HornForm';
 
 
 class App extends React.Component{
   constructor(props){
     super(props);
     this.state={
       show: false,
-      beastNumber: 0
+      beastNumber: 0,
+      hornNumber: 0
     }
   }
-
+  handleChange = event => {
+    this.setState({hornNumber: event.target.value});
+    this.setState({hornNumber: event.target.value});
+    this.setState({hornNumber: event.target.value});
+    this.setState({hornNumber: event.target.value});
+    this.setState({hornNumber: event.target.value});
+  }
```

Comment:
Any time you have lots of repetitive you should be suspicious of what you just wrote. Find a way to cut this down.
I'm also sort of unsure why you're wanting to setState in the exact same way five times...

Reply:
Oh dang I'm embarassed lol, this was for debugging because I was getting a really weird error and I got frustrated because this line wasn't going through at the right time so I put it 5 times... meant to clean it up and forgot, thanks for pointing it out